docs/guides/using-beluga-amcl.md

```diff
@@ -1,3 +1,153 @@
+
+- **Software Requirements:**
+  - **OS:** you can use either Ubuntu 22.04 LTS (Jammy Jellyfish) or Ubuntu 20.04 LTS (Focal Fossa).
+  - **ROS 2:** you can use either ROS 2 Humble Hawksbill or ROS 2 Jazzy Jalisco.
+
+:::{important}
+Most localization and SLAM packages in ROS 2 that take 2D LiDAR scans expect them published as `sensor_msgs/LaserScan` messages over the `/scan` topic. Likewise, these packages also expect odometry broadcasted as `odom` to `base_link` transforms over `tf`. That is the case for Beluga AMCL as well. Make sure your system follows this convention before using Beluga AMCL, or take the necessary steps to [reconfigure Beluga AMCL](../packages/beluga_amcl/docs/ros2-reference.md#parameters) to match your own conventions.
+:::
+
+## Setting things up
+
+1. **Install Beluga** by following [its installation guide](../getting-started/installation).
+
+2. **Install additional packages** like [SLAM Toolbox](https://github.com/SteveMacenski/slam_toolbox) and the [Nav2 `map_server`](https://github.com/ros-navigation/navigation2/tree/main/nav2_map_server):
+
+     ```bash
+     sudo apt install ros-${ROS_DISTRO}-slam-toolbox ros-${ROS_DISTRO}-map-server
+     ```
+
+   You will need these for mapping work.
+
+:::{tip}
+Remember to source your ROS 2 environment to use it:
+
+```bash
+source /opt/ros/${ROS_DISTRO}/setup.bash
+```
+:::
+
+## Mapping the environment
+
+Before the robot can localize using Beluga AMCL, we need a map of the environment. Here's how you can create one:
+
+1. **Launch SLAM Toolbox**, a widely adopted SLAM system:
+
+   ```bash
+   ros2 launch slam_toolbox online_async_launch.py
+   ```
+
+2. **Drive the robot around**, and make sure you cover all the areas you want included in the map. As the robot moves, SLAM Toolbox will incrementally build a map. You can visualize this using Rviz:
+
+   ```bash
+   ros2 run rviz2 rviz2
+   ```
+
+    Make sure to add the `Map` display and point it to the `/map` topic using a transient local durability policy.
+
+3. **Save the map** when you are satisfied with coverage:
+
+   ```bash
+   ros2 run nav2_map_server map_saver_cli -f ~/map
+   ```
+
+   This will create two files: `map.yaml` and `map.pgm`, which represent the map's metadata and image, respectively.
+
+## Configuring Beluga AMCL
+
+Let’s put everything together in one cohesive configuration.
+
+```bash
+editor beluga.launch.xml
+```
+
+Here’s a sample `launch` file in XML format you can use:
+
+```xml
+<launch>
+    <arg name="map_path" description="Path to you map.yaml file." />
+
+    <!-- Load map -->
+    <node pkg="nav2_map_server" exec="map_server" name="map_server" output="screen">
+        <param name="yaml_filename" value="$(var map_path)" />
+    </node>
+
+    <!-- Start AMCL -->
+    <node pkg="beluga" exec="amcl" name="amcl" output="screen">
+        <param name="use_sim_time" value="false" />
+        <param name="min_particles" value="500" />
+        <param name="max_particles" value="2000" />
```

Comment:
Most of these are defaults, can we omit them?

Reply:
Yes we can omit them all. Done in https://github.com/Ekumen-OS/beluga/pull/447.